OpenKh.Kh2/Battle/Fmlv.cs

```diff
@@ -26,44 +27,50 @@ public enum FormFm
             AntiForm
         }
 
-        public class Level
+        [Data] public byte Unk0 { get; set; }
+        [Data] public byte Unk1 { get; set; }
```

Comment:
May be a good idea to name these 2 fields, since they are not "unknown"

Reply:
It's true that they're not unknowns per-se, but those fields are only read and written internally. The actual use case is written below.